go.mod

```diff
@@ -100,6 +100,7 @@ require (
 	github.com/dustin/go-humanize v1.0.0 // indirect
 	github.com/dvsekhvalnov/jose2go v1.5.0 // indirect
 	github.com/edsrzf/mmap-go v1.0.0 // indirect
+	github.com/evmos/evmos/v13 v13.0.0-20230421203240-b08b5c2bcd96 // indirect
```

Comment:
To remove this dependency that is breaking the tests we need to update evmos-ledger-go to use evmos v14, and then update the import here